app/main/views/index.py

```diff
@@ -35,6 +44,11 @@ def privacy():
     return render_template("views/privacy.html")
 
 
+@main.route("/robots.txt")
+def robots():
```

Comment:
I would just call this static() rather than robots().  That way when we get other files which may need to be sent this way you can just add the `@main.route("/other-file")` and the name is still accurate